### PR TITLE
Avoid search collapse after page refresh

### PR DESCRIPTION
**Pull request by surinder-tsys**
**Actions Performed**
1. Open the app and login
2. Now open the menu
3. Go to another area
4. Open the menu again
5. Go back to the main files page
6. Select the search
7. Start entering a search term

**Expected Result**
The search remains being properly opened.

**Actual Result**
The search now closes incorrectly after a short period of time.

- [ ] Tests written, or not not needed

### CODE REVIEW

**Comment by tobiasKaminsky** [comment]
From my test this does not solve it, as after refresh is finished entire OCFileList is being redrawn and thus search collapses.

**Comment by surinder-tsys** [comment]
> From my test this does not solve it, as after refresh is finished entire OCFileList is being redrawn and thus search collapses.

Hi @tobiasKaminsky 
We will review it and will update the PR.

**Comment by surinder-tsys** [comment]
> From my test this does not solve it, as after refresh is finished entire OCFileList is being redrawn and thus search collapses.

Fixed and PR updated. Please check.

**Review by ZetaTom — APPROVED** [review]

File: app/src/main/java/com/owncloud/android/ui/fragment/OCFileListFragment.java [hunk 1]
```diff
@@ -1337,23 +1339,6 @@ public void listDirectory(OCFile directory, OCFile file, boolean onlyOnDevice, b
                         return; // no files, wait for sync
                     }
                 }
```
Comment (ZetaTom) [inline app/src/main/java/com/owncloud/android/ui/fragment/OCFileListFragment.java:1341]:
```suggestion
                }

```

**Pull request merged**